packages/hash/integration/src/tests/integration.test.ts

```diff
@@ -1,3 +1,11 @@
+/**
+ * @todo: re-implement integration tests on the GraphQL layer
```

Comment:
A little concerned about this but I think it makes sense to do

Reply:
It will be a big task, but something we are going to have to do anyways now that we are refactoring the GraphQL type definitions so definitely not something we should address before that.